### DISCUSSION TITLE
Frient EMIZB-141 Energy Meter Energy Sensor Unknown

### DISCUSSION THREAD

**Discussion by pigioldHAOS** [Koenkk/zigbee2mqtt#19999] [Devices]
Morning,
Installed and configured on HA I am not getting the energy data in kWh, while the power and battery status are ok.
Does anyone have the same problem? 

Thanks

**Comment by MUlisse** [comment]
I do. Also looking for a fix.

**Reply by nivek1612** [reply]
same here. No energy reading

**Reply by pigioldHAOS** [reply]
It's a shame it can't be resolved
{
    "battery": 100,
    "battery_low": false,
    "linkquality": 54,
    "power": 457,
    "voltage": 3100,
    **"energy": null**
}

**Reply by nivek1612** [reply]
There is a way 
https://github.com/Koenkk/zigbee2mqtt/issues/19042

**Comment by nivek1612** [comment]
Check this out https://github.com/Koenkk/zigbee2mqtt/issues/19042

**Reply by pigioldHAOS** [reply]
Thank you. I'll try to understand something since I don't know it very well yet.
